pepper-apis/studybuilder-cli/studies/lms/i18n/es.json

```diff
@@ -41,6 +41,34 @@
       "required_hint": "Please choose a province"
     }
   },
+
+  "somatic_results": {
```

Comment:
we don't have es language support but added since translation file exists. No impact as of now